src/ray/gcs/tables.cc

```diff
@@ -1,7 +1,65 @@
 #include "ray/gcs/tables.h"
 
+#include "ray/gcs/client.h"
+
+#include "task.h"
+#include "common_protocol.h"
```

Comment:
Nit: if you install clang-format support for your editor, these will be auto re-ordered :)